charts/hono/values.yaml

```diff
@@ -350,7 +350,7 @@ adapters:
       # exposed AMQP endpoints.
       # If not set, the adapter by default exposes the secure and insecure ports
       # using an example key and certificate.
-      amqp:
+      amqp: {}
```

Comment:
For consistency, this change could also be made to the `authServer.hono.auth.amqp` field below, adapting the `hono-service-auth-secret.yaml` accordingly.